internal/service/login.go

```diff
@@ -24,71 +22,83 @@ type AuthenticationApi interface {
 
 // Login Logs into grafana returning a legacyClient instance using Token or Basic SecureData
 func (s *DashNGoImpl) Login() {
+	var err error
+	userInfo, err := s.GetUserInfo()
```

Comment:
It doesn't look like we're doing anything with the error here. If we fail to login that's probably a fatal error?

Reply:
Well, that's a valid behavior, GetUserInfo() could fail if the user is token based, so if the user is using basic auth and happens to be an admin then we'll set the value otherwise it's a NoOp which is fine.  I updated the doc a bit to reflect the intended behavior.